cmd/rpc/web/wallet/components/util.js

```diff
@@ -663,3 +663,37 @@ export async function retryWithDelay(fn, onFailure, retries = 8, delayMs = 1000,
         }
     }
 }
+
+// getActionFee() returns the fee for a given action based on the params
+export function getActionFee(action, params) {
+  switch (action) {
+      case "send":
+          return params.sendFee;
+      case "stake":
+          return params.stakeFee;
+      case "create_order":
+          return params.createOrderFee;
+      case "lock_order":
+          return params.lockOrderFee;
```

Comment:
There is no lock order fee parameter - was this tested?

Reply:
I did test them but still interesting, I think that I read about it on the params shown in the explorer (which I got the info from), seems like I read wrong as I was adding them on my side, fixing